README.md

```diff
@@ -0,0 +1,47 @@
+# seniormap
+An interactive map of GMHS seniors' college/postsecondary destinations.  Created by Erik Boesen (GMHS Class of 2019), the map is now maintained the GMHS Advanced Programming club. If you'd like to make a submission to the map please navigate to https://apc-gm.com/seniormap/ and click on the form in the bottom right corner. 
+
+## Maintainence instructions 
+### Creating a the form and spreadsheet for a new year
+
+1. Navigate to the 'Senior Map' folder on Google Drive and create a new folder with the year of the graduating class (e.g. 2027)
+2. Make a copy of a previous year's form and edit content accordingly for the appropriate year
+3. Ensure the spreadsheet has a [linked Google spreadsheet](https://drive.google.com/file/d/1aFtfqwHGrV9CzMXrnqTfLghL3VVLddmU/view?usp=sharing) with appropriate fields
```

Comment:
Yeah embedding the images would look much cleaner and make the whole tutorial easier to understand. The images are being uploaded in this PR and after it gets merged I can then embed all the images directly by doing: 

`![alt text](https://github.com/[username]/[reponame]/blob/[branch]/image.jpg?raw=true)`

I could do them beforehand by uploading the Imgur or something but that's gross -- I just don't know what the URL will be for the images

Reply:
You don't have to do that. Just do
```
![alt](tutorial_images/tutX.png)
```
It will work without the full link; I've done it before successfully.

Also, I would prefer the directory of images be called `readme`.

Reply:
Oh, nice - always learning new things! I'll make both changes